src/main/java/org/spongepowered/common/mixin/api/mcp/entity/monster/EndermanEntityMixin_API.java

```diff
@@ -24,20 +24,13 @@
  */
 package org.spongepowered.common.mixin.api.mcp.entity.monster;
 
-import org.spongepowered.api.data.manipulator.mutable.entity.ScreamingData;
+import net.minecraft.entity.monster.EndermanEntity;
 import org.spongepowered.api.entity.living.monster.Enderman;
 import org.spongepowered.asm.mixin.Mixin;
 
 import java.util.Collection;
-import net.minecraft.entity.monster.EndermanEntity;
 
 @Mixin(EndermanEntity.class)
 public abstract class EndermanEntityMixin_API extends MonsterEntityMixin_API implements Enderman {
 
-    @Override
-    public void spongeApi$supplyVanillaManipulators(Collection<? super Mutable<?, ?>> manipulators) {
```

Comment:
Missing IS_SCREAMING Value in api$getVanillaValues()

Reply:
Will probably have to add an api method to get the Value.Mutable<Boolean> from the enderman.

Reply:
Isn't it a Key?

Reply:
It is a key but don't we want it to be accessible as a `Value.Mutable<Boolean>` on the enderman